packages/theme/src/styles/default-theme.css

```diff
@@ -34,15 +34,16 @@
   --swui-font-weight-link: 400;
   --swui-font-buttons: var(--swui-font-primary);
   --swui-font-input: var(--swui-font-primary);
+  --swui-line-height: 1.5;
   --swui-font-size-inputs: 1.4rem;
   --swui-font-size-large: 1.6rem;
-  --swui-line-height-large: 2.4rem;
+  --swui-line-height-large: var(--swui-line-height);
```

Comment:
Is this correct? This will decrease the line height?

Reply:
`line-height: 1.5` means:
font-size: 1.6rem -> line-height: 2.4rem (prev 2.4)
font-size: 1.4rem -> line-height: 2.1rem (prev 1.6)
font-size: 1.2rem -> line-height: 1.8rem (prev 1.6)
font-size: 1.0rem -> line-height: 1.5rem (prev 1.6)

So you're correct in that the smallest font size will have a slightly smaller line height than before. The others will be larger or unchanged.

Reply:
I don't think this is a problem, we rarely use small font-size for text where this is relevant?